frontend/src/metabase/visualizations/components/ObjectDetail/utils.unit.spec.ts

```diff
@@ -12,22 +15,57 @@ import {
 } from "metabase-types/api/mocks/presets";
 import Question from "metabase-lib/Question";
 
+import Database from "metabase-lib/metadata/Database";
 import {
-  getObjectName,
+  getActionItems,
   getDisplayId,
   getIdValue,
+  getObjectName,
   getSinglePKIndex,
+  isValidImplicitDeleteAction,
+  isValidImplicitUpdateAction,
 } from "./utils";
 
 const card = createSavedStructuredCard({
   name: "Special Order",
 });
 
+const database = createSampleDatabase();
+
+const databaseWithEnabledActions = new Database({
```

Comment:
Creating metabase-lib entities directly is forbidden. The correct way is to use `createMockMetadata` and get them via `metabase.database(...)`. Unfortunately there is nothing preventing people from calling the constructor.

Reply:
b0b7a3ac68d9d5752defe5d815a3f3e1d7f3930b